src/js/modules/wallet/controllers/settings/security/settings.security.ctrl.js

```diff
@@ -0,0 +1,512 @@
+                                    body_class: 'text-center',
+                                    ok: false
+                                });
+
+                                return formSettingsService.updateLaunchService2FA($scope.isEnabled2fa)
+                                    .then(function () {
+                                        isEnabled2fa = $scope.isEnabled2fa;
+
+                                        launchService.updateAccountInfo({
+                                            requires2FA: false
+                                        }).then(function () {
+                                            accountSecurityService.updateSecurityScore();
+                                            $timeout(function() {
+                                                pleaseWaitDialog.dismiss();
+                                            }, 1500);
+                                        });
+                                    });
+                            }, function (e) {
+                                // Error handler for wrong two factor token
+                                $scope.isEnabled2fa = isEnabled2fa;
+
+                                if (pleaseWaitDialog) {
+                                    pleaseWaitDialog.dismiss();
+                                }
+
+                                dialogService.alert({
+                                    title: $translate.instant('SETTINGS_2FA'),
+                                    body: e.message || e
+                                });
+                            });
+                    }, function() {
+                        // Reset for enter two factor token
+                        $scope.isEnabled2fa = isEnabled2fa;
+                    });
+            });
+    }
+
+    $scope.verifyEmail = function () {
+        var sdk = activeWallet.getSdkWallet().sdk;
+
+        if (!settings.verifiedEmail) {
+            return sdk.requestVerificationEmail()
+                .then(function (result) {
+                    // err handling
+                    if (!result) {
+                        return dialogService.alert(
+                            $translate.instant('ERROR_TITLE_2'),
+                            $translate.instant('MSG_ERROR_SENDING_MAIL')
+                        ).result;
+                    }
+
+                    return dialogService.alert(
+                        $translate.instant('CHECK_YOUR_INBOX'),
+                        $translate.instant('MSG_EMAIL_VERIFY')
+                    ).result
+                })
+                .catch(function (e) {
+                    return dialogService.alert(
+                        $translate.instant('ERROR_TITLE_2'),
+                        $translate.instant('MSG_ERROR_SENDING_MAIL')
+                    ).result;
+                });
+        } else {
+            return dialogService.alert(
+                "Nothing to do",
```

Comment:
Add to translations and replace with translation token

Reply:
thanks, missed this one